Library/Styles/BaseStyles.swift

```diff
@@ -81,6 +81,18 @@ public func dropShadowStyle <V: UIViewProtocol> (radius: CGFloat = 2.0,
       <> V.lens.layer.shadowOffset .~ offset
 }
 
+public func dropShadowStyleMedium <V: UIViewProtocol> () -> ((V) -> V) {
+  return dropShadowStyle(radius: 5.0, offset: .init(width: 0, height: 2.0))
+    <> V.lens.layer.shadowOpacity .~ 0.17
+    <> V.lens.layer.shadowColor .~ UIColor.black.cgColor
+}
+
+public func dropShadowStyleLarge <V: UIViewProtocol> () -> ((V) -> V) {
+  return dropShadowStyle(radius: 6.0, offset: .init(width: 0, height: 3.0))
+    <> V.lens.layer.shadowOpacity .~ 0.17
+    <> V.lens.layer.shadowColor .~ UIColor.black.cgColor
+}
```

Comment:
just a note: adding these medium and large styles in for future work. Soon, we'll be updating the base drop shadow style with the black color and lighter opacity so we can remove these last 2 lines from these functions once that's done.